### PR TITLE
Create JSONHelper.lua

### PR DESCRIPTION
**Pull request by jdahlblom**
Wondering how to isolate it more in regards of JSON.lua.

### CODE REVIEW

File: Scripts/DCS-BIOS/lib/common/JSONHelper.lua [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	if file then
+		file:write(json_string)
+		file:close()
+		file = nil
+	end
+end
+
+--- @func Encodes to JSON and writes to file
+--- @param filename string File name including path
+--- @return any
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/lib/common/JSONHelper.lua:27]: return type should be specified (it's a `table`, I believe? `{ [string]: any }` maybe?)

File: Scripts/DCS-BIOS/lib/common/JSONHelper.lua [hunk 2]
```diff
@@ -0,0 +1,47 @@
+module("JSONHelper", package.seeall)
+
+--- @class JSONHelper
+local JSONHelper = {}
+
+--- @func Encodes to JSON and writes to file
+--- @param value any
+--- @param filename string File name including path
+function JSONHelper.encode_to_file(value, filename)
+	local JSON = BIOS.json
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/lib/common/JSONHelper.lua:10]: The only place BIOS.json will be used is in this file, right? I'm thinking at the very least we should get rid of BIOS.json and just keep that definition local to this file.

**Review by charliefoxtwo — COMMENTED** [review]
Looks like stylua wasn't run on this file (likely not running on any files in this folder).

It _may_ be possible for us to just use `require("JSON")`, not sure.

**Comment by jdahlblom** [comment]
> Looks like stylua wasn't run on this file (likely not running on any files in this folder).
I will add it

**Pull request merged**